src/components/layout/dashboard/contextHeader/ContextHeaderContent.tsx

```diff
@@ -14,7 +14,7 @@ const ContextHeaderContent: React.FC<Props> = ({ breadcrumbs }) => {
       alignItems='center'
       justifyContent='space-between'
       flex={1}
-      sx={{ pl: 3 }}
+      sx={{ pl: 1 }}
```

Comment:
It looks like if we want this to always line up with the dashboard content, we will need to make the padding dynamic to match:
https://github.com/greenriver/hmis-frontend/blob/48aba505f964f852e6c44d22e6479ae662575588/src/components/layout/dashboard/DashboardContentContainer.tsx#L72

<img width="1792" height="1330" alt="Screenshot 2025-09-08 at 8 20 06 PM" src="https://github.com/user-attachments/assets/a49f6231-9a32-4d51-9954-c56e13e561d5" />